### PR TITLE
10-37: Add ability to exclude logs from find matching non-logrotated filed in /var/log

### PR DESCRIPTION
**Pull request by Daenou**

### CODE REVIEW

File: plugins/modules/find.py [hunk 1]
```diff
@@ -16,11 +16,16 @@
 
 # If this is part of a collection, you need to use semantic versioning,
 # i.e. the version is of the form "2.5.0" and not "2.4".
-version_added: "0.1.0"
+version_added: "0.1.1"
```
Comment (s3lph) [inline plugins/modules/find.py:19]: This field defines the version of the collection in which this module was introduced. It should not be updated.
```suggestion
version_added: "0.1.0"
```

File: plugins/modules/find.py [hunk 2]
```diff
@@ -16,11 +16,16 @@
...
 
 description: Wrapper around the UNIX find utility, because ansible.builtin.file capabilities are very restricted.
 
 options:
+    pattern:
+        description: Regex that the files must match
```
Comment (s3lph) [inline plugins/modules/find.py:25]: Not a regex

```suggestion
        description: Pattern that the files must match (through find -name ...)
```

**Review by s3lph — CHANGES_REQUESTED** [review]

File: plugins/modules/find.py [hunk 3]
```diff
@@ -16,11 +16,16 @@
 
 # If this is part of a collection, you need to use semantic versioning,
 # i.e. the version is of the form "2.5.0" and not "2.4".
```
Comment (Daenou) [inline plugins/modules/find.py]:
```suggestion
# version_added describes the collection version in which this module was added, don't adjust this value
```

**Review by s3lph — APPROVED** [review]
LGTM 🚀

**Pull request merged**